packages/utils/recursive-parser.ts

```diff
@@ -15,7 +15,7 @@ export default function recursiveTransform(
 		console.error("Error in parsing top scope, Array required");
 		return false;
 	} else if (key === "merge") {
-		if (typeof value === "string") {
+		if (typeof value[0] === "string" && Array.isArray(value)) {
```

Comment:
both indices are strings here, start by checking array, then that both indices are strings